test/data/schemas.test.ts

```diff
@@ -187,6 +192,21 @@ test('state incentives JSON files match schemas', async tap => {
           `must define city attribute on corresponding authority ${incentive.authority} for incentives with city authority type`,
         );
       }
+      if (incentive.authority_type === AuthorityType.Other) {
+        tap.hasProp(
+          incentive,
+          'geo_group',
+          `authority_type 'other' must include a geo group (id ${incentive.id})`,
+        );
+
+        const stateGroups = GEO_GROUPS_BY_STATE[stateId];
```

Comment:
Performance isn't a concern, but I'd move this up to where we define `authorities` since it's the same for every record in each state.